extensions/amp-story/0.1/amp-story-system-layer.js

```diff
@@ -127,21 +146,39 @@ export class SystemLayer {
 
     this.isBuilt_ = true;
 
-    this.root_ = renderAsElement(this.win_.document, TEMPLATE);
+    this.root_ = this.win_.document.createElement('div');
+    const shadowRoot = createShadowRoot(this.root_);
+
+    this.systemLayerEl_ = renderAsElement(this.win_.document, TEMPLATE);
+
+    const style = this.win_.document.createElement('style');
```

Comment:
(not necessarily in this PR) Seems like we should create a util like `addCssToNode(shadowRoot, CSS)`, since this is a pattern we'll use in all of our components, right?

Reply:
Good idea! Let's add this to our utils in a followup PR